go/signedexchange/cmd/gen-signedexchange/main.go

```diff
@@ -78,13 +79,23 @@ func run() error {
 		return fmt.Errorf("failed to read private key file %q. err: %v", *flagPrivateKey, err)
 	}
 
-	parsedPrivKey, _ := pem.Decode(privkeytext)
-	if parsedPrivKey == nil {
-		return fmt.Errorf("invalid private key")
+	var privkey crypto.PrivateKey
+	for {
+		var parsedPrivKey *pem.Block
+		parsedPrivKey, privkeytext = pem.Decode(privkeytext)
+		if parsedPrivKey == nil {
+			return fmt.Errorf("invalid private key")
```

Comment:
`fmt.Errorf("Invalid PEM block in private key file %q.", *flagPrivateKey)`

Reply:
Done.